rpcs3/Emu/RSX/Common/ranged_map.hpp

```diff
@@ -128,7 +128,14 @@ namespace rsx
 		public:
 			bool operator == (const iterator& other) const
 			{
-				return m_it == other.m_it;
+				// Iterators are only valid if the related data is valid

```

Comment:
This really doesn't make sense. What exactly is asserting?

Reply:
For reference: you can't compare two iterators if they are not iterators of the same object.
You can only compare iterators of the same object, or e.g. two default iterators.